Update DDR4 OC Guide.md

LunarPSD opened the pull request:
Many grammar and text corrections as well as updated information, links, and easier readability.

LunarPSD committed:
Update DDR4 OC Guide.md

Many grammar and text corrections as well as updated information, links, and easier readability.

LunarPSD committed:
Update DDR4 OC Guide.md

Unlinked and embedded some images, fixed broken link, and fixed typos,

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -37,24 +37,21 @@
   * [Extreme config by anta777](https://drive.google.com/file/d/1uegPn9ZuUoWxOssCP4PjMjGW9eC_1VJA) (recommended). Make sure to load the config. It should say 'Customize: Extreme1 @anta777' if loaded.  
   Credits: [u/nucl3arlion](https://www.reddit.com/r/overclocking/comments/dlghvs/micron_reve_high_training_voltage_requirement/f4zcs04/).
-  * [Here](https://www.overclock.net/threads/memory-testing-with-testmem5-tm5-with-custom-configs.1751608/) is a link to TM5 pre-packaged with many configs.
+  * [Here](https://drive.google.com/drive/u/0/folders/1E-5ZsrlKwr1SlEWxK3iLebms2k3EkQc) is a link to TM5 pre-packaged with many configs.
```
Why did you change the URL? It doesn't seem to work anyway

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -37,24 +37,21 @@
   * Note that AVX and SSE can vary in error detection speed. On Intel-based systems, SSE appears better for testing IMC voltages while AVX appears better for DRAM voltage.
-  * The Large AVX2 CPU test is a great stability test for your CPU and RAM at the same time. The more you tune your ram the harder it'll be to stable in this test.
+  * The Large AVX2 CPU test is a great stability test for your CPU and RAM at the same time. The more you tune your ram the harder it'll be to stable in this test. Be sure to run Normal Mode as Extreme will not use as much RAM.
+  * The VRAM test at maximum utilization in conjunction with Prime95 Large FFTs will stress FCLK and is recommended when testing FCLK stability.
```
Has anyone else verified this?

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -94,39 +91,46 @@ You should always test with a variety of stress tests to ensure your overclock i
-* [Intel](https://i.imgur.com/hcKDkCc.png)
-* [AMD](https://i.imgur.com/Ie4LVtI.png)
-* RAM timings are split into 3 categories: primary, secondary and tertiary. These are indicated by 'P', 'S', and 'T' respectively.
+* Intel
+
+  ![image](https://user-images.githubusercontent.com/69487009/156914502-104b0cbd-bbc4-4c94-a583-426e84f732eb.png)
+
+* AMD
+
+  ![image](https://user-images.githubusercontent.com/69487009/156914518-5510c6a2-6187-41ee-8acd-a8f76a5590c4.png)
+
```
Why did you change the URLs?

LunarPSD replied:
This is just to embed the images.

integralfx replied:
What do you mean by embed?

LunarPSD replied:
People won't have to click the link in order to see the images and it'll appear in the document as an image. You can see this in preview mode.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -170,42 +183,42 @@ Sometimes the Thaiphoon report won't tell you the IC or it may misidentify the I
-   * x16 configurations will have half as many banks and bank groups compared to the traditional x8 configurations which means less performance. See [buildzoid's video](https://www.youtube.com/watch?v=k6SIdxq2yxE) for more information.
-* Density matters when determining how far your ICs can go. For example, 4Gb AFR and 8Gb AFR will not overclock the same despite sharing the same name. The same can be said for Micron Rev. B which exists as both 8Gb and 16Gb. The 16Gb ICs overclock better and are sold as both in 16GB and 8GB capacities despite the DIMMs using 8 chips. The 8GB sticks have their SPD modified and can be found in hig ...
-* As the count of ranks total in a system increases, so does the load on the memory controller. This usually means that more memory ranks will a require higher voltage, especially VCCSA on Intel and SOC voltage on AMD.
+   * x16 configurations will have half as many banks and bank groups compared to the traditional x8 configurations which mean less performance. See [buildzoid's video](https://www.youtube.com/watch?v=k6SIdxq2yxE) for more information.
```
Pretty sure 'means' is the correct word here

LunarPSD replied:
I agree.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -170,42 +183,42 @@ Sometimes the Thaiphoon report won't tell you the IC or it may misidentify the I
-* As the count of ranks total in a system increases, so does the load on the memory controller. This usually means that more memory ranks will a require higher voltage, especially VCCSA on Intel and SOC voltage on AMD.
+   * x16 configurations will have half as many banks and bank groups compared to the traditional x8 configurations which mean less performance. See [buildzoid's video](https://www.youtube.com/watch?v=k6SIdxq2yxE) for more information.
+* Density matters when determining how far your ICs can go. For example, 4Gb AFR and 8Gb AFR will not overclock the same despite sharing the same name. The same can be said for Micron Rev. B which exists as both 8Gb and 16Gb. The 16Gb ICs overclock better and are sold both in 16GB and 8GB capacities despite the DIMMs using 8 chips. The 8GB sticks have their SPD modified and can be found in higher ...
+* As the count of ranks' total in a system increases, so does the load on the memory controller. This usually means that more memory ranks will require higher voltage, especially VCCSA on Intel and SOC voltage on AMD.
```
```suggestion
* As the total count of ranks in a system increases, so does the load on the memory controller. This usually means that more memory ranks will require higher voltage, especially VCCSA on Intel and SOC voltage on AMD.
```

I think this makes more sense

LunarPSD replied:
I agree, it has been added.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -251,7 +264,7 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
   Manufacturers would separate ICs into different containers/bins depending on their frequency. Hence the term binning.
 * G.Skill is one manufacturer known for extensive binning and categorization. Oftentimes, several different SKUs of G.Skill memory will belong to the same factory bin (i.e. DDR4-3600 16-16-16-36 1.35V bin of B-Die being binned the same as DDR4-3200 14-14-14-34 1.35V B-Die).
-* B-die binned for 2400 15-15-15 is significantly worse than good B-die binned for DDR4-3200 14-14-14 or even DDR4-3000 14-14-14. Don't expect it to have the same voltage scaling characteristics as good B-Die.
+* B-die binned for 2400 15-15-15 is significantly worse than good B-die binned for DDR4-3200 14-14-14 or even DDR4-3000 14-14-14. Don't expect it to have the same voltage scaling characteristics as a good B-Die.
```
It should be either

> as good B-Die

or

> as a good B-Die kit

LunarPSD replied:
This has been fixed.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -263,28 +276,28 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
+* The tRFC timings are very dependent on temperatures, as they are related to capacitor leakage which is affected by temperature. Higher temperatures will need higher tRFC values. tRFC2 and tRFC4 are timings that activate when the operating temperature of DRAM hits 85°C. Below these temperatures, these timings don't do anything.
 * B-Die is temperature sensitive and its ideal range is ~30-40°C. Some may be able to withstand higher temperatures so YMMV.
-* Rev. E, on the other hand, doesn't seem to be as strongly temperature sensitive, demonstrated by [buildzoid](https://www.youtube.com/watch?v=OeHEtULQg3Q).
+* Rev. E, on the other hand, doesn't seem to be as strongly temperature-sensitive, demonstrated by [buildzoid](https://www.youtube.com/watch?v=OeHEtULQg3Q).
```
I don't think the hyphen is necessary here

LunarPSD replied:
I realize that is correct.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -302,33 +318,33 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
+* SOC voltage is the voltage to the IMC and like with Intel, it's not recommended to leave it on auto. Typical ranges for this value range around 1.00V and 1.10V. Higher values are generally acceptable and may be necessary for stabilizing higher capacity memory and may aid in attaining FCLK stability.
 * By contrast, when SOC voltage is set too high, memory instability can occur. This negative scaling typically occurs between 1.15V and 1.25V on most Ryzen CPUs.
-  > There are clear differences in how the memory controller behaves on the different CPU specimens. The majority of the CPUs will do DDR4-3466 or higher at 1.050V SoC voltage, however the difference lies in how the different specimens react to the voltage. Some of the specimens seem scale with the increased SoC voltage, while the others simply refuse to scale at all or in some cases even illustr ...
+  > There are clear differences in how the memory controller behaves on the different CPU specimens. The majority of the CPUs will do DDR4-3466 or higher at 1.050V SoC voltage, however, the difference lies in how the different specimens react to the voltage. Some of the specimens seem to scale with the increased SoC voltage, while the others simply refuse to scale at all or in some cases even ill ...
```
This is a quote so it's not necessary to fix the grammar mistakes

LunarPSD replied:
It has been fixed.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -302,33 +318,33 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
-Likewise if you have VDDG set to 1.100V and start increasing the SoC voltage, your VDDG will raise as well. I don't have the exact figure, but you can assume that the minimum drop-out voltage (Vin-Vout) is around 40mV.
-Meaning you ACTUAL SoC voltage has to be at least by this much higher, than the requested VDDG for it to take effect as it is requested.  
+  > Most cLDO voltages are regulated from the two main power rails of the CPU. In the case of cLDO_VDDG and cLDO_VDDP, they are regulated from the VDDCR_SoC plane.
+Because of this, there are a couple of rules. For example, if you set the VDDG to 1.100V, while your actual SoC voltage under load is 1.05V the VDDG will stay roughly at 1.01V max.
+Likewise, if you have VDDG set to 1.100V and start increasing the SoC voltage, your VDDG will raise as well. I don't have the exact figure, but you can assume that the minimum drop-out voltage (Vin-Vout) is around 40mV.
+Meaning your ACTUAL SoC voltage has to be at least by this much higher, than the requested VDDG for it to take effect as it is requested.  
 Adjusting the SoC voltage alone, unlike on previous gen. parts doesn't do much if anything at all.
-The default value is fixed 1.100V and AMD recommends keeping it at that level. Increasing the VDDG helps with the fabric overclocking in certain scenarios, but not always.
+The default value is fixed at .1.100V and AMD recommends keeping it at that level. Increasing the VDDG helps with the fabric overclocking in certain scenarios, but not always.
 1800MHz FCLK should be doable at the default 0.9500V value and for pushing the limits it might be beneficial to increase it to =< 1.05V (1.100 - 1.125V SoC, depending on the load-line).  
```
This is a quote so it's not necessary to fix the grammar mistakes

LunarPSD replied:
Fixed.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -382,54 +398,63 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
-   On AMD, start off with 1.10V SOC.
+1. * Ensure your sticks are in the recommended DIMM slots (usually 2 and 4).
+   * Make sure your CPU overclock is disabled when tuning RAM, as an unstable CPU can lead to memory errors. When pushing high frequency with tight timings, your CPU may become unstable and may need to be re-done.
+   * Make sure your UEFI/Bios is up to date.
```
```suggestion
   * Make sure your UEFI/BIOS is up to date.
```

LunarPSD replied:
This suggestion has been committed.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -382,54 +398,63 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
+     * Note that some boards have auto rules that can stifle your progress, an example being tCWL = tCL - 1 which can lead to uneven values of tCWL. Reading the [Miscellaneous Tips](#miscellaneous-tips) might give you insight into your particular platform and your motherboard's functionality.
 1. Intel:
-   * Increase VCCSA and VCCIO to 1.25V.
+   * Increase VCCSA and VCCIO to 1.25V. For ADL, VCCIO does not exist.
```
If you're going to mention ADL, add all the necessary info about it (gear 1, IMC voltages and their daily maximums, etc.)

LunarPSD replied:
Information about those has been added, as well as daily maximums are not fully known due to it being new.

LunarPSD committed:
Update DDR4 OC Guide.md

Co-authored-by: integralfx <16512539+integralfx@users.noreply.github.com>

LunarPSD committed:
Update DDR4 OC Guide.md

Co-authored-by: integralfx <16512539+integralfx@users.noreply.github.com>

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Merge branch 'oc-guide' into patch-1

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

Removed broken links

LunarPSD committed:
Update DDR4 OC Guide.md

LunarPSD committed:
Update DDR4 OC Guide.md

Grammar and readability corrections

LunarPSD committed:
Update DDR4 OC Guide.md

Removed some TM5 files as they will be added directly to the repo.

LunarPSD committed:
Update DDR4 OC Guide.md

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -37,30 +37,28 @@
-  * [LMHz Universal 2 config](https://www.hardwareluxx.de/community/threads/ryzen-ram-oc-m%C3%B6gliche-limitierungen.1216557/page-159#post-27506598)
-  * If you experience issues with all threads crashing upon launch with the extreme config it might help to edit the row "Testing Window Size (Mb)=1408". Replace the window size with your total RAM (minus some margin for Windows) divided by your processors available threads (e.g. 12800/16 = 800 MB per thread).
-* [OCCT](https://www.ocbase.com/) with the dedicated memory test using either SSE or AVX instructions.
-  * Note that AVX and SSE can vary in error detection speed. On Intel-based systems, SSE appears better for testing IMC voltages while AVX appears better for DRAM voltage.
-  * The Large AVX2 CPU test is a great stability test for your CPU and RAM at the same time. The more you tune your ram the harder it'll be to stable in this test.
+  * [Absolut](https://github.com/integralfx/MemTestHelper/blob/oc-guide/absolutnew.cfg)
+  * [PCBdestroyer](https://github.com/integralfx/MemTestHelper/blob/oc-guide/PCBdestroyer.cfg)
+  * [LMHz Universal 2 config](https://github.com/integralfx/MemTestHelper/blob/oc-guide/Universal-2%40LMhz.cfg)
```
These configs don't exist in the repo

You need to add it to your repo and use a relative link
e.g. `TM5/extreme%40anta777.cfg`

LunarPSD replied:
I was thinking of the files being added to your repo directly but that may be too messy with all the files. I'll go do that.

integralfx replied:
You can't add the files directly to my repo anyway.

You need to add them to yours so that the changes get merged into my branch.

I'd recommend you read into git branching: https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

LunarPSD replied:
I have added relative links to all the files.

integralfx commented on DDR4 OC Guide.md:
```diff
@@ -375,62 +397,82 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
-3. Set primary timings to 16-20-20-40 (tCL-tRCD-tRP-tRAS) and tCWL to 16.
-   * Most ICs need loose tRCD and/or tRP which is why I recommend 20.
+   * ICs that are known to roll over above 1.35V include but are not limited to: 8Gb Samsung C-die, older Micron/SpecTek ICs (before 8Gb Rev. E).
+   * Here is a list of common ICs and safe voltages for them:
+   
+   | IC                                | Daily voltage (V) | Extreme voltage |
+   | :---:                             | :---------:       | :-------------: |
+   | 8Gbit Samsung B-Die               | 1.45 - 1.55       | 1.55 - 1.65     |
+   | 8Gbit Hynix DJR/16Gbit AJR        | 1.45 - 1.55       | 1.50 - 1.65     |
+   | 8Gbit Micron Rev. E/16Gbit Rev. B | 1.45 - 1.55       | 1.55 - 1.60     |
+   | 8Gbit Hynix CJR                   | 1.35 - 1.40       | 1.40 - 1.45     |
+   | 16Gbit Hynix CJR                  | 1.40 - 1.50       | 1.50 - 1.55     |
+   | 8Gbit Nanya B-Die                 | 1.35 - 1.40       | 1.40 - 1.45     |
+   | Samsung C-Die                     | 1.30 - 1.35       | Likely Unsafe   |
```
This should go under Maximum Recommended Daily Voltage

Clarify what extreme voltage means

LunarPSD replied:
That makes sense, I will also add a link to the section in the part where they overclock.

LunarPSD replied:
This has been moved and clarified.

integralfx commented:
**It's up to the starter of the conversation to resolve the conversation.** 

You should leave a comment saying that you agree/disagree. If you disagree, provide a reason.

Don't just close the conversation without saying anything. That's just a huge middle finger to the starter of the conversation, especially if you keep repeatedly doing this (without even addressing the issue).

LunarPSD committed:
Fixed broken TM5 link

LunarPSD committed:
Add files via upload

LunarPSD committed:
Added configs and corrected voltage values/grammar

LunarPSD committed:
Moved tWR to the end

tWR is a timing that most people skip/save for last, so it would make sense to put it near the end instead of the beginning.
